docs/source/howto/installation.rst

```diff
@@ -558,13 +558,13 @@ In order to restore a backup, you will need to:
 
 Managing multiple users
 =======================
-Setups with multiple users for a single AiiDA instance are currently not supported.
-Instead, each AiiDA user should install AiiDA in a Unix/Windows account on their own computer.
-Under this account it is possible to configure all the credentials necessary to connect to remote computers.
-Using independent accounts will ensure that, for instance, the SSH credentials to connect to supercomputers are not shared with others.
+AiiDA currently does not support multiple users running concurrently on the same AiiDA instance (database & file repository).
+While AiiDA will tag any node with the :py:class:`~aiida.orm.users.User` who created it, there is currently no permission system in place to limit the operations that can be performed by a given user.
 
-Data can be shared between instances using :ref:`AiiDA's export and import functionality <how-to:share:archives>`.
-Sharing (subsets of) the AiiDA graph can be done as often as needed.
+AiiDA assumes that each user has AiiDA installed on the operating system account of their computer.
```

Comment:
Not quite sure what this sentence is supposed to convey.

Reply:
This comes from the previous version of the text; I think it was meant as a practical suggestion to have people install aiida in their unix account (otherwise one could imagine having AiiDA installed by root & different users just running on separate profiles).

I will rephrase this a bit; but would also be ok to remove it entirely.